RocketChat/Rocket.Chat
# fix: importer filter not working

## Proposed changes (including videos or screenshots)

The importer filter is not working, inserting all records from the file and ignoring the filtering (`do_import` property).
Also, it considers the total amount of users in the file instead of only the selected ones to check if there are enough seats available.




## Issue(s)


## Steps to test or reproduce
1. Go to `admin > sidebar > importer`;
2. Select a file to import and click on `import`;
3. Select only a few users and channels;
4. Start the import process;
5. Go to `admin > users` and check that all users from the file were imported;
6. Go to `admin > rooms` and check that all channels from the file were imported;

If using an enterprise license, the import process will fail if the file contains more users than seats available, even if only one user is selected to import.


## Further comments

[WM-160](https://rocketchat.atlassian.net/browse/WM-160)

[WM-160]: https://rocketchat.atlassian.net/browse/WM-160?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
